roles/addons/diskless/files/disklessset.py

```diff
@@ -263,7 +300,7 @@ def generate_ipxe_boot_file(image_type, image_name, image_kernel, image_initramf
             os.system('rm -Rf /var/www/html/preboot_execution_environment/diskless/images/'+selected_image_name)
             os.system('mkdir /var/www/html/preboot_execution_environment/diskless/images/'+selected_image_name)
             print(bcolors.OKBLUE+'[INFO] Generating new ipxe boot file.'+bcolors.ENDC)
-            boot_file_content = generate_ipxe_boot_file('nfs_staging', selected_image_name, kernel_list[int(selected_kernel)], 'initramfs-kernel-'+kernel_list[int(selected_kernel)].strip('vmlinuz-'))
+            boot_file_content = generate_ipxe_boot_file('nfs_staging', selected_image_name, kernel_list[int(selected_kernel)], 'initramfs-kernel-'+kernel_list[int(selected_kernel)].strip('vmlinuz-'), '')
```

Comment:
For now it is ok this way, we can add selinux support on NFS later. Ok for me.

Reply:
ok

Reply:
If `selinux` defaults to False, no need to change the call to generate_ipxe_boot_file() here.